docs/react-view-models.md

```diff
@@ -8,17 +8,17 @@
 
 `react-view-models` is a library to connect observable view-models to [React](https://facebook.github.io/react/) [presentational components](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0#.v9i90qbq8) to create auto rendering [container components](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0#.v9i90qbq8).
 
-- The [react-view-models.connect connect] method takes a [DefineMap](./can-define/map/map.html) class and a React component class, and returns a new React Component which will autorender when obsevable changes happen on the view model
+- The [react-view-models.connect connect] method takes a [DefineMap](./can-define/map/map.html) class and a React component class, and returns a new React Component which will auto-render when observable changes happen on the view-model
 
 @body
 
 React-View-Models follows the pattern popularized by [react-redux](https://github.com/reactjs/react-redux), and provide users with a [react-view-models.connect connect] function for extending [React](https://facebook.github.io/react/) Presentational Components into Container Components, by providing `connect` with either a `mapToProps` function or more commonly a `ViewModel` constructor function, which is an extended [DefineMap](./can-define/map/map.html) class, along with a presentational component, just like react-redux does.
 
-The `ViewModel` is an observable, and when any observable change happens to one of it's properties, or if new props get set on the Container Component, it will be serialized and sent into the connected component as props, forcing an update/render.
+The `ViewModel` is an observable, and when any observable change happens to one of it's properties, or if new props get set on the Container Component, some of the view-models properties will be sent into the connected component as props, forcing an update/render.
```

Comment:
Cool thanks @marshallswain ! It's great to have someone be an editor for documentation.